src/Components/Header/Header.js

```diff
@@ -80,7 +81,7 @@ const Header = ({ handleTextfieldChange }) => {
             <Select
               labelId="select-language-label"
               id="select-language"
-              value={context.locale}
+              value={userLanguage in languageOptions ? userLanguage : context.locale}
```

Comment:
This does not change the language, it just changes what the select is showing. To set the language, update the `Wrapper.tsx` file, and have the default language on line 103 be set to local storage if it exists, or to the navigator language.